beta-src/src/utils/api/index.ts

```diff
@@ -41,9 +41,9 @@ export const submitOrders = (
 ): Promise<AxiosResponse> => {
   if (Object.keys(queryParams).length) {
     return orderSubmission.post(
-      `/ajax.php?${buildQueryString(queryParams)}`,
+      `ajax.php?${buildQueryString(queryParams)}`,
```

Comment:
same should these be `/ajax.php`

Reply:
No because axios automatically adds the trailing slash